Simplify meeting demos to leverage externalUserId in roster

aburnson opened the pull request:
*Description of changes*

Simplify meeting demos to leverage externalUserId in roster

- Build roster from external user ID
- Clean up server.js and serverless, add comments, and remove unnecessary APIs (/meeting /attendee)
- Use innerText instead of innerHTML in meeting V2 demo
- Move pause button to bottom right and merge with resume
- Consolidate serverless handlers into one file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

ltrung commented on demos/browser/app/meeting/meeting.ts:
```diff
@@ -711,12 +712,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
-            const name = json.AttendeeInfo.Name;
-            this.roster[attendeeId].name = name ? name : '';
-          }
+          this.roster[attendeeId].name = externalUserId.split('#')[1];
```
Do we need to do the same changes like in meetingV2?

aburnson replied:
There is a similar change in meetingV2:

```
      if (!this.roster[attendeeId]) {
        this.roster[attendeeId] = {
          name: externalUserId.split('#')[1] + (isContentAttendee ? ' «Content»' : ''),
        };
      }
```

I did the most extensive cleanup to `meetingV2` and only fixed `meeting` for backwards compatibility hence the smaller and slightly different change set in `meeting`.

asnyatkov commented on demos/browser/server.js:
```diff
@@ -1,153 +1,105 @@
+          MediaRegion: requestUrl.query.region,
+          // Any meeting ID you wish to associate with the meeting.
+          // For simplicity here, we use the meeting title.
+          ExternalMeetingId: requestUrl.query.title,
```
`requestUrl.query.title.substring(0, 64);` to avoid 400 in case the title is too long.

aburnson replied:
Fixed

asnyatkov commented on demos/serverless/deploy.js:
```diff
@@ -116,6 +116,7 @@ function ensureApp(appName) {
 
   // TODO: remove this once AWS Lambda Node.js runtime includes the Chime APIs
```
+1 please remove aws-sdk copying step.

aburnson replied:
I have generalized this into a list of packages to be included at the top of the deploy script. I would like to continue to include `aws-sdk` here so we can show usage with the latest API (including `CreateMeeting:ExternalMeetingId` which was added recently).

asnyatkov commented on demos/serverless/deploy.js:
```diff
@@ -137,6 +138,8 @@ if (!fs.existsSync('build')) {
 console.log(`Using region ${region}, bucket ${bucket}, stack ${stack}`);
 ensureBucket();
 
+fs.copySync(path.join(__dirname, '..', 'browser', 'node_modules', 'aws-sdk'), 'src');
```
ChimeSDK has to be available in all regions and lambdas by now.

aburnson replied:
I have generalized this into a list of packages to be included at the top of the deploy script. I would like to continue to include `aws-sdk` here so we can show usage with the latest API (including `CreateMeeting:ExternalMeetingId` which was added recently).

aburnson commented:
> Seems like Travis failed to launch the demo app in local host (returned 404).

Yes, I removed the `/meeting` endpoint since it does virtually the same thing as `/join` except doesn't create an attendee. I've updated the integration test to use `/join` and create a dummy user.
